codec-http/src/main/java/io/netty/handler/codec/http/multipart/AbstractDiskHttpData.java

```diff
@@ -114,21 +114,18 @@ public void setContent(ByteBuf buffer) throws IOException {
             }
             if (buffer.readableBytes() == 0) {
                 // empty file
-                if (!file.createNewFile()) {
-                    throw new IOException("file exists already: " + file);
-                }
                 return;
             }
             FileOutputStream outputStream = new FileOutputStream(file);
             try {
-                FileChannel localfileChannel = outputStream.getChannel();
+                FileChannel localFileChannel = outputStream.getChannel();
```

Comment:
please refer this change as it is not related to the "fix"

Reply:
Picked into separate commit.